cli/crates/typed-resolvers/tests/schema_types/basic.expected.ts

```diff
@@ -1,3 +1,16 @@
+// This is a generated file. It should not be edited manually.
+//
+// You can decide to commit this file or add it to your `.gitignore`.
+//
+// By convention, this module is imported as `@grafbase/generated`. To make this syntax possible,
+// add a `paths` entry to your `tsconfig.json`.
```

Comment:
Should we add this automatically?

Reply:
Yeah we definitely could, but I'm a bit worried about messing with people's tsconfig.

Reply:
We do that for `package.json` already...